tests/benchdnn/inputs/matmul/test_matmul_gpu

```diff
@@ -60,6 +60,12 @@
 --attr-post-ops=sum+add:f32+add:u8:per_dim_01+mul:f32:per_dim_0+add:s8:per_tensor+add:f32:per_dim_01+linear:2:-1
 3x30x1:3x1x20
 
+--reset
+--dt=s8:u8:f16
+--bia-dt=f16
+--attr-scales=src0:per_tensor:f16:1x32
+--batch=shapes_3d_bcast
```

Comment:
Generally, consolidating sizes in a batch file is not helping as dimensions are tied to 32 and changing sizes there incompatible with group_size=32 would lead to a problem spot here. I suggest to keep specific problems of interest here instead of a batch file for simpler adjustments if they are needed.